crates/turbopack-node/src/evaluate.rs

```diff
@@ -96,8 +104,49 @@ pub async fn evaluate(
         ),
     };
     emit(bootstrap.cell().into(), intermediate_output_path).await?;
-    let pool = NodeJsPool::new(cwd, entrypoint, HashMap::new(), 1);
+    let pool = NodeJsPool::new(
+        cwd,
+        entrypoint,
+        HashMap::new(),
+        available_parallelism().map_or(1, |v| v.get()),
+    );
+    Ok(pool.cell())
+}
+
+#[turbo_tasks::function]
+/// Pass the file you cared as `runtime_entries` to invalidate and reload the
+/// evaluated result automatically.
+pub async fn evaluate(
+    context_path: FileSystemPathVc,
+    module_asset: AssetVc,
+    cwd: FileSystemPathVc,
+    context: AssetContextVc,
+    intermediate_output_path: FileSystemPathVc,
+    runtime_entries: Option<EcmascriptChunkPlaceablesVc>,
+    args: Vec<JsonValueVc>,
+) -> Result<JavaScriptValueVc> {
+    let pool = get_evaluate_pool(
+        context_path,
+        module_asset,
+        cwd,
+        context,
+        intermediate_output_path,
+        runtime_entries,
+    )
+    .await?;
     let mut operation = pool.operation().await?;
-    let output = eval_js_operation(&mut operation, EvalJavaScriptOutgoingMessage::Evaluate).await?;
+    let args = args.into_iter().map(|v| v).try_join().await?;
```

Comment:
Clippy is warning about:
```suggestion
    let args = args.into_iter().try_join().await?;
```